### PR TITLE
DDF-1274: reorganized DDF Catalog documentation.

### PR DESCRIPTION
**Pull request by ricklarsen**
@oscaritoro @stustison @vinamartin @clockard @djblue @gordocanchola @ahoffer

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/453" target="_blank">ReviewNinja

File: catalog/docs/src/main/resources/IntContents.adoc [hunk 1]
```diff
@@ -192,23 +201,365 @@ Transforms also work on read operations for metacards in federated sources.
...
+
+Install the `catalog-transformer-json` feature using the Admin Console.
+
+*Configuration*
+
+None
+
+*Implementation Details*
+
+.HEY CHECK THIS TABLE
```
Comment (vinamartin) [inline catalog/docs/src/main/resources/IntContents.adoc]: :question:

File: catalog/docs/src/main/resources/IntContents.adoc [hunk 2]
```diff
@@ -192,23 +201,365 @@ Transforms also work on read operations for metacards in federated sources.
...
+The Catalog Transformers App will install this feature when deployed.
+This transformer's feature, `catalog-transformer-metadata`, can be uninstalled or installed.
+
+*Configuration*
+
+None
+
+*Implementation Details*
+
+.HEY CHECK THIS TABLE TOO
```
Comment (vinamartin) [inline catalog/docs/src/main/resources/IntContents.adoc]: This table needs to be checked, as well. 
It _longs_ to be checked.

File: catalog/docs/src/main/resources/IntContents.adoc [hunk 3]
```diff
@@ -560,102 +920,110 @@ The OpenSearch Description Document is an XML file is found inside of the OpenSe
...
-====== Example Output
-
-The default output for OpenSearch is Atom.  Detailed documentation on Atom output, including query result mapping and example output, can be Atom Query Response Transformer found on the page.
-
 ====== Known Issues
-None
 
-== Developing a New Endpoint
+None
+// move to Extending...
```
Comment (vinamartin) [inline catalog/docs/src/main/resources/IntContents.adoc]: Address this?

**Comment by vinamartin** [comment]
LGTM :+1:

File: catalog/docs/src/main/resources/IntContents.adoc [hunk 4]
```diff
@@ -192,23 +201,365 @@ Transforms also work on read operations for metacards in federated sources.
...
+
+Install the `catalog-transformer-html` feature using the Admin console.
+
+*Configuration*
+
+None
+
+*Implementation Details*
+
+.HEY CHECK THIS TABLE
```
Comment (clockard) [inline catalog/docs/src/main/resources/IntContents.adoc]: Check this table and remove?

File: catalog/docs/src/main/resources/IntContents.adoc [hunk 5]
```diff
@@ -192,23 +201,365 @@ Transforms also work on read operations for metacards in federated sources.
...
+
+The HTML Metacard Transformer is responsible for translating a Metacard into an HTML formatted document.
+
+*Usage*
+
+Using the REST Endpoint, for example, request a metacard with the transform option set to the HTML shortname.
+
+[source,http]
+----
+http://localhost:8181/services/catalog/0123456789abcdef0123456789abcdef?transform=html
```
Comment (clockard) [inline catalog/docs/src/main/resources/IntContents.adoc]: Should this be changed https://localhost:8993 or maybe use the <HOST>:<PORT> notation? If you change this change it in the rest of the document as well.

Comment (ahoffer) [inline catalog/docs/src/main/resources/IntContents.adoc]: In a separate ticket, we might want to create a variable for default port, like we did for branding.

**Comment by clockard** [comment]
After comments are addressed, LGTM :+1:

File: catalog/docs/src/main/resources/IntContents.adoc (hunk 1 above)
Comment (ricklarsen) [inline catalog/docs/src/main/resources/IntContents.adoc]: oops. Well, did you check the table?

File: catalog/docs/src/main/resources/IntContents.adoc (hunk 2 above)
Comment (ricklarsen) [inline catalog/docs/src/main/resources/IntContents.adoc]: seems legit.

File: catalog/docs/src/main/resources/IntContents.adoc (hunk 3 above)
Comment (ricklarsen) [inline catalog/docs/src/main/resources/IntContents.adoc]: fixed.

File: catalog/docs/src/main/resources/IntContents.adoc (hunk 4 above)
Comment (ricklarsen) [inline catalog/docs/src/main/resources/IntContents.adoc]: yes. will do.

File: catalog/docs/src/main/resources/IntContents.adoc (hunk 5 above)
Comment (ricklarsen) [inline catalog/docs/src/main/resources/IntContents.adoc]: Yes, and good idea.

File: catalog/docs/src/main/resources/IntContents.adoc [hunk 6]
```diff
@@ -168,7 +176,8 @@ Note that for all RESTful CRUD commands only one metacard ID is supported in t
 
 ===== Interacting with the REST CRUD Endpoint
 
-Any web browser can be used to perform a REST read. Various other tools and libraries can be used to perform the other HTTP operations on the REST endpoint (e.g., soapUI, cURL, etc.)
+Any web browser can be used to perform a REST read.
+Various other tools and libraries can be used to perform the other HTTP operations on the REST endpoint (e.g., soapUI, cURL, etc.)
 
 ===== Metacard Transforms with the REST CRUD Endpoint
```
Comment (oscaritoro) [inline catalog/docs/src/main/resources/IntContents.adoc]: Are the "Example:" lines bellow rendering correctly?

Comment (ricklarsen) [inline catalog/docs/src/main/resources/IntContents.adoc]: fixed.

**Comment by oscaritoro** [comment]
Ship it :+1:

**Pull request merged**